src/components/Forms/MultiAmountSelector.vue

```diff
@@ -115,14 +117,21 @@ export default {
 	},
 	watch: {
 		customAmountModel(newVal) {
-			// emit a custom amount updated event only if valid
-			if (!this.$v.$invalid) {
-				// Implementation Required in Parent Component
-				// Enables tracking of custom option/key associated value
-				this.$emit('custom-amount-updated', newVal);
-			}
+			// Implementation Required in Parent Component
+			// Enables tracking of custom option/key associated value
+			this.$emit('custom-amount-updated', newVal);
 		},
 		selected(newVal) {
+			// to focus custom amount input element
+			if (newVal === 'custom') {
+				this.$nextTick(() => {
+					try {
+						this.$refs.kvCurrencyRef[0].$refs.kvCurrencyInputRef.focus();
```

Comment:
@ryan-ludwig This has the affect of focusing the the input and placing the blinking cursor alongside the number. The focus event to `select` the text is applied to the parent component above and only works when you click into the input field AND if the input is already present. I tried adding an extra `select()` event here too but it had no effect.

Reply:
An overall improvement but it got really messy to add that focus event :(

Reply:
Yeah, this select is acting finicky. I was able to make the focus happen (but not select oddly) using
```
				this.$nextTick(() => {
					try {
						this.$refs.kvCurrencyRef[0].$el.select();
					} catch (e) { ...
```

Reply:
@mcstover When I select "custom", the input is populating with $5 for me. I don't think we want to anchor the user at the smallest amount possible. Maybe we should anchor it to $25 and let them modify from there, or set it to whatever was selected before they hit custom? I think best case is to have it empty, but I'm not sure how we get around the validation rules. @eddieferrer do you have any thoughts?

Reply:
I don't like the idea of hacking around the validation rules.. I'm fine with having custom start at whatever is selected before they hit custom

Reply:
Focus was already working fine when clicking the custom option  though I see now that it's not setting focus when you switch between onetime/recurring if you previously had the custom option selected. Getting it to select the text isn't required if the focus stuff is working ok.

Tiff and Olly will instruct on what the default should be when the custom option is chosen. I don't think we should leave it empty but if they wanted to do that it would be fine to just let the validation rule show it's error as useful instruction for the valid range that needs to be input.

Reply:
Ok latest push adds focus functionality when switching between the onetime/recurring components if the custom option was selected prior. Focus was already in place when click the "Other" box within the components.

I'll follow up during testing to see what if Olly + Tiff want default amounts and what they should be for each deposit type if so.